ansible/roles/notebooks/files/requirements.txt

```diff
@@ -1,5 +1,6 @@
 nbgrader==0.6.1
 yellowbrick==1.1
+seaborn=0.10.1
```

Comment:
Hello! This dependency is actually already included in the base image, which is based on the [jupyter/datascience-notebook](https://github.com/jupyter/docker-stacks/blob/master/datascience-notebook/Dockerfile) image which, in turn, is based on the [jupyter/scipy-notebook](https://github.com/jupyter/docker-stacks/blob/master/scipy-notebook/Dockerfile) image.